src/styles/blocks/header.scss

```
@@ -33,10 +33,17 @@
   }
 
   &__link {
+    transition: $transition-duration;
```

```suggestion
    transition-duration: $transition-duration;
```
You don't need to use shorthand property now as it controls only duration.
Or you can set both of them like this:
`transition: color $transition-duration`